### ISSUE TITLE
Updated files for the Github Repo

### ISSUE THREAD

**Issue by atulnarayanan** [a-M-i-T/socket.io-live-auction-demo#2]
Hi, 
Your Heroku app seems to be running with the updated code which includes your todo list. Can you update the github repository to match the Github Repo?

**Comment by robertodevs** [comment]
Yes please, I tried to replicate this locally but does not work as the Heroku one. Thank you for your great contribution.

**Comment by aryaitandmedia** [comment]
@DeStiNi10 @robertodevs  There are few minor fixes, code optimization and dependencies check to be done. The codes are not well managed as I was just playing around with nodejs and socket.io back then. I will update all the files as soon as possible. Please check back later. Thank you.

**Comment by a-M-i-T** [comment]
@DeStiNi10 @robertodevs There are few minor fixes, code optimization and dependencies check to be done. The codes are not well managed as I was just playing around with nodejs and socket.io back then. I will update all the files as soon as possible. Please check back later. Thank you.

**Comment by faiakak** [comment]
Hi, can you please update the code on GIT or share the updated code as heroku version
